### ISSUE TITLE
Bug: `fallbackSort` causes `sort-imports` failed to fix

### ISSUE THREAD

**Issue by ntnyq** [azat-io/eslint-plugin-perfectionist#483]
### Repro steps

- Run `pnpm run lint` will report errors.

- Run `pnpm run lint:fix` error still exists.

- Comment `fallbackSort` option in `eslint.config.mjs`, no error reported.

Besides, if there is no `// eslint-disable-next-line perfectionist/sort-imports`, it still works fine.

### Reproduction repo

https://github.com/ntnyq-dev/repro-perfectionist-sort-imports

### Config

```ts
// @ts-check

import pluginPerfectionist from 'eslint-plugin-perfectionist'
import { config, configs } from 'typescript-eslint'

export default config(
  {
    extends: configs.recommended,
  },
  {
    plugins: {
      perfectionist: pluginPerfectionist,
    },
    rules: {
      '@typescript-eslint/ban-ts-comment': 'off',
      'perfectionist/sort-imports': [
        'error',
        {
          fallbackSort: { order: 'asc', type: 'alphabetical' },
          order: 'asc',
          type: 'alphabetical',
          groups: [
            // Side effect style imports (e.g. 'normalize.css')
            'side-effect-style',

            // Styles (e.g. *.{css,scss,less})
            'style',
            /**
             * Imports that don’t fit into any other group
             */
            'unknown',
          ],
          internalPattern: ['^~/.+', '^@/.+', '^#.+'],
        },
      ],
    },
  },
)
```

### Code

```ts
import '@unocss/reset/tailwind.css'
import 'element-plus/theme-chalk/dark/css-vars.css'
import 'element-plus/theme-chalk/index.css'
import 'uno.css'
// COMMENT BELLOW is IMPORTANT
// eslint-disable-next-line perfectionist/sort-imports
import '@/styles/app.scss'

```

### ESLint version

v9.22.0

### ESLint Plugin Perfectionist version

v4.10.0

### Additional comments

Since option `order` and `type` is the same as  option `fallbackSort`.

I think there is no need to attempt to apply fallback sort.

Maybe some deep equal check.

### Validations

- [x] Read the [docs](https://perfectionist.dev).
- [x] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by hugop95** [comment]
@ntnyq Thank you for the details, a fix will come soon. It's actually unrelated to `fallbackSort`.

Minimal reproduction case:

```ts
interface Test {
  b: string;
  c: string;
  // eslint-disable-next-line
  a: string;
}
```

**Comment by azat-io** [comment]
Fixed in 1ac83c9e64167e56616e1b4b49204f502d7f4fd5.
Released in [v4.10.1](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v4.10.1).
